Fix type casting due to missing header.

```
../src/graphics/FontReader.cpp: In function 'bool InitFontData()':
../src/graphics/FontReader.cpp:40:52: error: 'uint32_t' does not name a type
   40 |                 auto codePoint = *reinterpret_cast<uint32_t *>(ptr) & 0xFFFFFFU;
      |                                                    ^~~~~~~~
../src/graphics/FontReader.cpp:5:1: note: 'uint32_t' is defined in header '<cstdint>'; this is probably fixable by adding '#include <cstdint>'
    4 | #include "font.bz2.h"
  +++ |+#include <cstdint>
```

--- a/src/graphics/FontReader.cpp
+++ b/src/graphics/FontReader.cpp
@@ -4,6 +4,7 @@
 #include "font.bz2.h"
 
 #include <array>
+#include <cstdint>
 
 unsigned char *font_data = nullptr;
 unsigned int *font_ptrs = nullptr;
